docs/development/from-another-module.md

```diff
@@ -0,0 +1,8 @@
+
+# Installation from another module
+
+To use this code in another SDK module, call `kb-sdk install SampleService` in the other module's root directory.
```

Comment:
We may need to update this since it isn't a dynamic service any longer and this could result in getting an old client.

Reply:
Yes, I left that and some other old docs in there. I did try to fix some clearly obsolete information. In this case I wasn't sure if it was still valid to use kb-sdk to install a client, or if that is the most common practice, compared to using a generic client, so left that in place. It can certainly be removed at any time if it is invalid.

Reply:
Also, I didn't read whole swaths of existing content for accuracy or completeness, just moved them around.